src/evict/evict_lru.c

```diff
@@ -1577,10 +1609,10 @@ __evict_walk(WT_SESSION_IMPL *session, WT_EVICT_QUEUE *queue)
         }
         btree->evict_walk_skips = 0;
 
-        (void)__wt_atomic_addi32(&dhandle->session_inuse, 1);
-        incr = true;
+        /* Remember the file to visit */
```

Comment:
nit: This comment doesn't add much context
```suggestion
```

Reply:
Agree.